src/pcapi/notifications/push/transactional_notifications.py

```diff
@@ -58,3 +64,15 @@ def get_tomorrow_stock_notification_data(stock_id: int) -> Optional[Transactiona
             body="Retrouve les détails de la réservation sur l’application pass Culture",
         ),
     )
+
+
+def get_offer_notification_data(user: User, offer: Offer) -> Optional[TransactionalNotificationData]:
+    return TransactionalNotificationData(
+        group_id=GroupId.OFFER_LINK.value,
+        user_ids=[user.id],
+        message=TransactionalNotificationMessage(
+            title=f"Ton lien pour réserver {offer.name}",
+            body=f"Tu as mis {offer.name} en favoris, clique ici pour finaliser ta réservation !",
+        ),
+        extra={"deeplink": offer_webapp_link(offer)},
```

Comment:
le terme "deeplink" est plutôt réservé pour des liens propres à l'app native, pas vraiment pour la webapp.
```suggestion
        extra={"offer_url": offer_webapp_link(offer)},
```

Reply:
c'est un champ de l'api batch @xordoquy cf https://doc.batch.com/api/transactional/send

Reply:
Soit j'ai raté l'option `offer_url` dans la doc de Batch, soit on ne parle pas de la même chose.

Reply:
C'est moi, j'ai tellement l'habitude d'associer ce genre de structure à l'API que je n'avais pas fait attention que c'était pour batch, désolé pour le bruit.